spec/presenters/content_items_csv_presenter_spec.rb

```diff
@@ -122,6 +122,10 @@
         expect(subject[3]).to start_with('http')
       end
 
+      it 'has correct Content Data Link' do
+        expect(subject[3]).to eq('http://content-data.dev.gov.uk/metrics')
+      end
```

Comment:
How about extending the scope of the preceding test, rather than adding a new one, as they seem quite similar?

Reply:
Sounds good, I'll update that now 👍

Reply:
Great :+1: